packages/core/README.md

```diff
@@ -286,3 +287,15 @@ Once you are connected to the SDK with your key and a user ID, you can check the
 ```typescript
 const isValid = await isTokenValid();
 ```
+
+### getUriLogFile
+
+```typescript
+  getUriLogFile(): Promise<{ uri: string } | null>;
+```
+
+Once you are connected to the SDK with your key and a user ID, you can check the validity of the generated token by calling:
```

Comment:
Proposition : `You can retrieve the Uri log file by calling the following method:`